frontend/src/pages/clusterSettings/ClusterSettings.scss

```diff
@@ -1,7 +1,20 @@
 .odh-cluster-settings {
-  .pf-c-form__group {
-    border: var(--pf-global--BorderWidth--sm) solid var(--pf-global--BorderColor--100);
-    padding: var(--pf-global--spacer--md);
-    margin: 0;
+    .pf-c-form__group {
+      border: var(--pf-global--BorderWidth--sm) solid var(--pf-global--BorderColor--100);
+      padding: var(--pf-global--spacer--md);
+      margin: 0;
+    }
+
+    .odh-number-input { 
+      max-width: 100px;
+    }
+
+    .pf-c-input-group {
+      padding-top: var(--pf-global--spacer--md);
+      padding-bottom: var(--pf-global--spacer--md);
+    }
+
+    &__form {
+      margin: 0 var(--pf-global--spacer--lg);
+    }
   }
```

Comment:
some problems with the indent and format of this file.

Reply:
Done.